app/assets/stylesheets/redde/redactor/index.scss

```diff
@@ -330,20 +327,20 @@ body .redactor_toolbar li a.redactor_btn_normalscreen span		  { background-posit
 	background-color: #eee;
   	-webkit-border-radius: 5px;
   	-moz-border-radius: 5px;
-  	border-radius: 5px;  	
+  	border-radius: 5px;
 }
 
 /* Drag and Drop Area */
 .redactor_droparea {
 	position: relative;
     width: 100%;
-    margin: auto;	
+    margin: auto;
     margin-bottom: 5px;
 }
-.redactor_droparea .redactor_dropareabox {    
+.redactor_droparea .redactor_dropareabox {
```

Comment:
Merge rule `.redactor_droparea .redactor_dropareabox` with rule on line 334<br>Selector `redactor_droparea` should be written in lowercase with hyphens<br>Selector `redactor_dropareabox` should be written in lowercase with hyphens